jupyterhub/jupyterhub/overlays/jupyterhub-idle-culler/deploymentconfig.yaml

```diff
@@ -11,6 +11,23 @@ spec:
       labels:
         app: jupyterhub-idle-culler
     spec:
+      initContainers:
+        - name: check-for-jh
+          image: 'quay.io/odh-jupyterhub/jupyterhub-img:v0.3.3'
```

Comment:
`image: 'quay.io/odh-jupyterhub/jupyterhub-img:v0.3.5'`

Should be increased to 0.3.5 like in this PR:  https://github.com/opendatahub-io/odh-manifests/commit/1fb9f35a934b5d8c9c92b7dc5c4488afd8d778b3